user_manual/pim/calendar.rst

```diff
@@ -1,3 +1,143 @@
 ======================
 Using the calendar app
 ======================
+
+.. note:: The calendar app is not enabled by default and needs to be installed
+          separately from our App Store. Please ask your Administrator for it.
+
+The Nextcloud Calendar app works similar to other calendar applications you can
+sync your Nextcloud calendars and events with.
+
+When you first access the Calendar app, a default first calendar will be
+pre-generated for you.
+
+.. figure:: ./images/calendar_application.png
+
+Managing your calendars
+-----------------------
+
+Import a Calendar
+~~~~~~~~~~~~~~~~~
+
+If you want to transfer your calendar and their respective events to your Nextcloud
+instance, importing is the best way to do so.
+
+.. figure:: ./images/calendar_settings.png
+
+1. Click on the settings-icon labeled with `Settings & Import` at the left-bottom.
+
+2. After clicking on `+ Import Calendar` you can select a calendar file from your
+local device to upload.
+
+3. The upload can take some time and depends on how big the calendar you import
+is.
+
+.. note:: The Nextcloud Calendar application only supports iCalendar-compatible
+          `.ics`-files, defined in RFC 2445.
```

Comment:
RFC 2445 is what we don't support 😉 That's the legacy standard, version 1

We support RFC 5545.